shared/reducers/tracker.js

```diff
@@ -400,6 +413,7 @@ export default function (state: State = initialState, action: Action): State {
           },
         }
       }
+      break
```

Comment:
nothing complains since this happens to be at the end, but lets put this in anyways